js/huobipro.js

```diff
@@ -610,6 +610,10 @@ module.exports = class huobipro extends Exchange {
         return {
             'info': response,
             'id': response['data'],
+            'symbol': symbol,
+            'type': type,
+            'side': side,
+            'status': 'open',
```

Comment:
I'd actually set this to undefined, because the order might as well be closed by the time we receive the response, so can't guarantee the order status.

Reply:
Will update it shortly